apps/root/src/pages/strategy-guardian-detail/strategy-management/deposit/hooks/useEarnDepositActions.tsx

```diff
@@ -603,14 +603,13 @@ const useEarnDepositActions = ({ strategy }: UseEarnDepositActionParams) => {
     if (!strategy || !activeWallet?.address) {
       return false;
     }
-    const wrappedProtocolToken = getWrappedProtocolToken(strategy.network.chainId);
     const position = strategy.userPositions?.find((userPosition) => userPosition.owner === activeWallet.address);
     const companionHasPermission =
       !isIncrease ||
       position?.permissions[contractService.getEarnCompanionAddress(strategy.network.chainId)]?.includes(
         EarnPermission.INCREASE
       );
-    return wrappedProtocolToken?.address === strategy?.asset.address && isIncrease && !companionHasPermission;
+    return isIncrease && !companionHasPermission;
```

Comment:
A create also uses permit2 right? Wouldn't that need companion too?

Reply:
No need for companion permissions on a position that does not exist yet :)

Reply:
damn right